pretrain/tinyllama.py

```diff
@@ -62,7 +62,10 @@ def setup(
     eval: EvalArgs = EvalArgs(interval=1000, max_iters=100),
 ):
     hparams = locals()
-    logger = choose_logger(io.out_dir, logger_name, name=f"pretrain-{model.name}", resume=resume)
+    data = TinyLlama() if data is None else data
+    config = Config(name="tiny-llama-1.1b") if model is None else model
+
+    logger = choose_logger(io.out_dir, logger_name, name=f"pretrain-{config.name}", resume=resume)
 
     strategy = FSDPStrategy(auto_wrap_policy={Block}, state_dict_type="full", sharding_strategy="HYBRID_SHARD")
```

Comment:
This needs to be None if devices == 1

Reply:
Done.
And in reverse, we need to support `devices="auto"` as a way to specify a default in the configs.